vowpalwabbit/core/tests/merge_test.cc

```diff
@@ -27,7 +27,7 @@ TEST(merge_tests, merge_simple_model)
   vw2->finish_example(*ex2);
 
   std::vector<const VW::workspace*> workspaces = {vw1.get(), vw2.get()};
-  auto result = VW::merge_models(workspaces);
+  auto result = VW::merge_models(nullptr, workspaces);
```

Comment:
maybe add `merge_with_base`

Reply:
Id like to avoid the rename for now, but will revisit later